docs/reference_guide.md

```diff
@@ -553,6 +554,11 @@ Default: `..`
 
 Trailer to add to strings that were truncated. Set to empty string to disable truncation trailers.
 
+### 9.11 `BPFTRACE_STACK_MODE`
```

Comment:
Please also update [bpftrace.adoc](https://github.com/iovisor/bpftrace/blob/master/man/adoc/bpftrace.adoc#environment-variables) . (We really have to unify it one of these days)